docs/tutorials/upgrading-airbyte.md

```diff
@@ -45,34 +49,34 @@ docker run --rm -v /tmp:/config airbyte/migration:0.20.0-alpha --\
 ```
 
 {% hint style="info" %}
-It may seem confusing that you need to specify the target version twice. The version passed as `--target-version` specifies the version to which the data will be migrated. Specifying the target version in the docker container tag makes sure that you are pulling an image that at least has the migration for the version you want. Technically the version used in the docker tag can be equal to or greater than the version you are upgrading to. For the simplicity of this tutorial we have them match.
+It may seem confusing that you need to specify the target version twice. The version passed as `--target-version` specifies the version to which the data will be migrated. Specifying the target version in the docker container tag makes sure that you are pulling an image that at least has the migration for the version you want. Technically the version used in the docker tag can be equal to or greater than the version you are upgrading to.
```

Comment:
Would it be less confusing if we just didn't suggest using `--target-version`? And then it would use the latest within the latest migration docker image?

Reply:
I put optional brackets and removed this hint, is that good?